packages/clerk-js/src/ui/common/alert/Alert.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export type AlertType = 'info' | 'error';
+
+export type AlertProps = {
+  type: AlertType;
+  children: React.ReactNode;
+  style?: {};
+} & React.HTMLAttributes<HTMLDivElement>;
+
+// Renders global alerts across components, will be replaced by notification snackbars.
+export const Alert: React.FC<AlertProps> = ({ type, children, style }) => {
```

Comment:
Because of https://spin.atomicobject.com/2022/01/04/think-twice-react-fc/ let's do the following:

```suggestion
export const function Alert({ type, children, style }: AlertProps { ... }
```

Reply:
(Somewhat unrelated to this PR, but still an interesting read)
React18 removes the implicit `children` prop from `React.FC` so in theory, we can use it in more places now. However, there are still valid reasons to avoid it: 

https://github.com/facebook/create-react-app/pull/8177#issue-537764080

Reply:
Converting to function